content/docs/command-reference/freeze.md

```diff
@@ -24,7 +24,7 @@ Freezing a stage is useful to avoid syncing data from the top of its
 (non-frozen) stages only.
 
 Note that <abbr>import stages</abbr> are frozen by default. Use `dvc update` to
-update the imported data from the external data source.
+bring the import up to date from the data source.
```

Comment:
with the data source?

Reply:
Hmmmm I see what you mean (using the data source) but "bring up to date" + "from <source>" seems better to me.